packages/libcs50-git/libcs50-git.pacscript

```diff
@@ -0,0 +1,21 @@
+pkgname="libcs50-git"
+gives="libcs50"
+pkgver="11.0.3"
+source=("https://github.com/cs50/libcs50.git")
+makedepends=("make")
+maintainer=("Vaibhav Sharma <vvaibhavv3434@gmail.com>")
+pkgdesc="Library provided by Harvard University for learning the basic C language"
+
+build() {
+  cd "${gives}"
+  make -j"${NCPU}"
+}
+
+package() {
+  cd "${gives}"
+  make DESTDIR="${pkgdir}/usr/local" install || true
```

Comment:
why is `|| true` being used here?

Reply:
Because of the ldconfig it presents in the make file when it's fail to run this command `ldconfig $(DESTDIR)/lib` it just gives error and the pacstall says package is not installed, so I added the `|| true` so it does not give error and I add the ldconfig command in the post_install function.

Reply:
And the rest of the script is just the cleanup like delete the build folder and stuff, so I can just ignore the error and move forward.

Reply:
Next time if I do a thing like that I just add a comment and that fine right, and you can just delete the comment later if you want to.

Reply:
> Because of the ldconfig it presents in the make file when it's fail to run this command `ldconfig $(DESTDIR)/lib` it just gives error and the pacstall says package is not installed, so I added the `|| true` so it does not give error and I add the ldconfig command in the post_install function.

i think i give the read only file system error ` [!] ERROR: Could not package libcs50-git properly
`.

Reply:
> > Because of the ldconfig it presents in the make file when it's fail to run this command `ldconfig $(DESTDIR)/lib` it just gives error and the pacstall says package is not installed, so I added the `|| true` so it does not give error and I add the ldconfig command in the post_install function.
> 
> i think i give the read only file system error `[!] ERROR: Could not package libcs50-git properly`.

and this also `/sbin/ldconfig.real: Can't create temporary cache file /etc/ld.so.cache~: Read-only file system
` by the make file.